server/storages/types.go

```diff
@@ -30,6 +30,8 @@ type Storage interface {
 	SyncStore(overriddenDataSchema *schema.BatchHeader, objects []map[string]interface{}, timeIntervalValue string) error
 	Update(object map[string]interface{}) error
 	Fallback(events ...*events.FailedEvent)
+	TestBatchProcessing(testName string, events []map[string]interface{}) error
```

Comment:
It's a good idea to have 1 Storage func for testing connection. 
I suggest to change this func signature to:
`TestBatchProcessing(eventCtx *adapters.EventContext) error` and call this func with prepared table(with name `jitsu_test_connection_${unique_identitifer}`) that has 1 test data object.

Reply:
After refactoring single `TestBatchProcessing` function exists. It does all needed operation with objects.